src/openqaoa-azure/pyproject.toml

```diff
@@ -1,3 +1,34 @@
 [build-system]
 requires = ["setuptools>=61.0"]
 build-backend = "setuptools.build_meta"
+
+[tool.setuptools.packages.find]
+where = ["."]
+
+[project]
+name = "openqaoa-azure"
+version = "0.2.5"
+authors = [{ name = "Entropica Labs" }]
+description = "Azure Plug-in for OpenQAOA"
+readme = "README.md"
+requires-python = ">=3.9, <3.12"
+license = { file = "LICENSE" }
+urls = { "Homepage" = "https://github.com/entropicalabs/openqaoa" }
+classifiers = [
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Operating System :: OS Independent",
+]
+keywords = ["quantum", "optimisation", "SDK"]
+
+dependencies = [
+    "openqaoa-core==0.2.5",
+    "openqaoa-qiskit==0.2.5",
```

Comment:
Same thing here, can this be fixed on the version of `openqaoa-azure` we are installing?